frontend/talentsearch/resources/js/components/skills/SkillResults.tsx

```diff
@@ -106,9 +106,7 @@ const SkillResults: React.FunctionComponent<SkillResultsProps> = ({
   const addedIds = addedSkills.map((skill) => skill.id);
   return (
     <section>
-      <h4>
-        {title} ({skills.length})
-      </h4>
+      <h4>{title}</h4>
```

Comment:
There is a [test for the title](https://github.com/GCTC-NTGC/gc-digital-talent/blob/feature/1683-prep-add-skills-to-experience/frontend/talentsearch/resources/js/tests/skills/SkillResults.test.tsx#L43-L49) that's currently failing. 

I guess were going to need a new prop called `totalSkills` for this instead. Do you want me to replace this with that?

Reply:
Good catch, thanks Yoni.  I've updated the test to expect the new behaviour.  I don't think we need a new prop though.  If the parent component wants to include the skills length in the title it can include that in the title prop.

Reply:
Oh ya good point 😆